service/api_task.go

```diff
@@ -291,10 +291,11 @@ func assignNextAvailableTask(taskQueue *model.TaskQueue, taskQueueService model.
 	var spec model.TaskSpec
 	if currentHost.LastTask != "" {
 		spec = model.TaskSpec{
-			Group:        currentHost.LastGroup,
-			BuildVariant: currentHost.LastBuildVariant,
-			ProjectID:    currentHost.LastProject,
-			Version:      currentHost.LastVersion,
+			Group:         currentHost.LastGroup,
+			BuildVariant:  currentHost.LastBuildVariant,
+			ProjectID:     currentHost.LastProject,
+			Version:       currentHost.LastVersion,
+			GroupMaxHosts: 0,
```

Comment:
nit: Delete this line, since it implies that 0 means something here, but really it's just the zero value of the integer type.

Reply:
Done.